src/components/cc-product-list/cc-product-list.stories.js

```diff
@@ -0,0 +1,614 @@
+        keywords: [],
+        iconUrl: 'https://docs.mailpace.com/img/logo.png',
+        url: '',
+        apiId: 'mailpace',
+      },
+    ],
+    icon: {
+      content: '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" version="1.1" style="shape-rendering:geometricPrecision;text-rendering:geometricPrecision;image-rendering:optimizeQuality;" viewBox="0 0 706.1 834.475" x="0px" y="0px" fill-rule="evenodd" clip-rule="evenodd"><defs><style type="text/css">.fil0 {fill:black;fill-rule:nonzero}</style></defs><g><path class="fil0" d="M353.05 0c69.01,0 124.97,55.95 124.97,124.97 0,60.82 -43.88,112.89 -103.68,123.15l0 89.24 138.47 100.6c19.64,-12.85 43.11,-20.32 68.32,-20.32 69.01,0 124.97,55.96 124.97,124.97 0,69.01 -55.96,124.97 -124.97,124.97 -69.01,0 -124.97,-55.96 -124.97,-124.97 0,-28.16 9.32,-54.15 25.04,-75.05l-128.15 -93.11 -128.15 93.11c15.72,20.9 25.04,46.89 25.04,75.05 0,69.01 -55.96,124.97 -124.97,124.97 -69.01,0 -124.97,-55.96 -124.97,-124.97 0,-69.01 55.96,-124.97 124.97,-124.97 25.21,0 48.69,7.47 68.32,20.32l138.46 -100.6 0 -89.24c-59.79,-10.26 -103.67,-62.33 -103.67,-123.15 0,-69.02 55.96,-124.97 124.97,-124.97zm0.37 208.13c34.59,0 82.01,-31.94 82.01,-83.16 0,-45.5 -36.88,-82.38 -82.38,-82.38 -45.5,0 -82.38,36.88 -82.38,82.38 0,51.89 42.51,83.16 82.75,83.16zm-228.45 252.1c-45.5,0 -82.38,36.88 -82.38,82.38 0,45.5 36.88,82.38 82.38,82.38 45.5,0 82.38,-36.88 82.38,-82.38 0,-45.5 -36.88,-82.38 -82.38,-82.38zm456.16 0c-45.5,0 -82.38,36.88 -82.38,82.38 0,45.5 36.88,82.38 82.38,82.38 45.5,0 82.38,-36.88 82.38,-82.38 0,-45.5 -36.88,-82.38 -82.38,-82.38z"/></g></svg>',
+    },
+  },
+];
+
+export const defaultStory = makeStory(conf, {
+  items: [
+    {
+      categoryDataList: PRODUCTS,
+    },
+  ],
+});
+
+export const filterInput = makeStory(conf, {
+  items: [
+    {
+      categoryDataList: PRODUCTS,
+      filterInput: 'cellar',
+    },
+  ],
+});
+
+export const irrelevantFilterInput = makeStory(conf, {
+  items: [
+    {
+      categoryDataList: PRODUCTS,
+      filterInput: 'irrelevant',
+    },
+  ],
+});
+
+export const filterCategory = makeStory(conf, {
+  items: [
+    {
+      categoryDataList: PRODUCTS,
+      filterCategory: 'Storage',
+    },
+  ],
+});
+
+export const irrelevantFilterCategory = makeStory(conf, {
+  items: [
+    {
+      categoryDataList: PRODUCTS,
+      filterCategory: 'irrelevant',
+    },
+  ],
+});
+export const irrelevantFilterCategoryButRelevantInput = makeStory(conf, {
+  items: [
+    {
+      categoryDataList: PRODUCTS,
+      filterCategory: 'irrelevant',
+      filterInput: 'sql',
+    },
+  ],
+});
+
+export const filterCategoryAndInput = makeStory(conf, {
+  items: [
+    {
+      categoryDataList: PRODUCTS,
+      filterCategory: 'Storage',
+      filterInput: 'cellar',
+    },
+  ],
+});
```

Comment:
issue: I think the story is outdated, the cellar product is no longer in the list, same for the storage category